addfunction.js

```diff
@@ -0,0 +1,354 @@
+let body=document.body
+let section=document.createElement('section')
+let i0=document.createElement('i')
+let ul=document.createElement('ul')
+let li0=document.createElement('li')
+let li1=document.createElement('li')
+let li2=document.createElement('li')
+let li3 = document.createElement('li')
+let a0 = document.createElement('a');
+let a1 = document.createElement('a');
+let a2 = document.createElement('a');
+let a3 = document.createElement('a');
+/*first project*/
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove the bunch of comments from your project to keep coder cleaner